shell/platform/darwin/ios/platform_view_ios.mm

```diff
@@ -171,14 +190,7 @@ new PlatformMessageHandlerIos(task_runners.GetPlatformTaskRunner())) {}
                         "PlatformViewIOS has no ViewController.";
     return;
   }
-  if (enabled && !accessibility_bridge_) {
```

Comment:
GitHub won't let me comment on the line above, but I think you're safe to delete the `if (!owner_controller_) { ... }` block now, since it's no longer used here.